src/main/java/io/chazza/advancementapi/AdvancementAPI.java

```diff
@@ -238,14 +252,13 @@ public String toString() {
         }
     }
 
-
     //BEGIN CLASSES
     @Builder(builderMethodName = "hiddenbuilder")
```

Comment:
This is useful but remember, this is a util class and it is not a good practice to force the user to install other dependencies on order to use it. I highly recommend to try and cut this and the bungee API from the code so all people need is one class.